src/coffee/import.coffee

```diff
@@ -498,7 +503,15 @@ class Import
       else
         Promise.resolve "[row #{rowIndex}] Product update not necessary."
 
+  assignState: (product) ->
```

Comment:
Could you rename this function to something like `assignDefaultState`? So it is more descriptive...

Reply:
But it's not assigning a default state; it's assigning a state.  I've changed the logged error message to make that more clear.

Reply:
If you call this function, it will always return `null` in case the `@states.key2id[@defaultState]` does not exist or ID of a state from `@defaultState` .. So that's why I thought that "default" should be in the function name because it does not return ID of any other state than the default one.

Reply:
Well, the return value is not used, but I can rename it `assignStateFromDefault`, which fixes the ambiguity of `assignDefaultState`.

Reply:
I am ok with that name ..